docs/trace/getting-started/README.md

```diff
@@ -75,7 +52,7 @@ Tracer](https://github.com/open-telemetry/opentelemetry-specification/blob/maste
 The activitysource instance is used to start an `Activity` which represent
 [OpenTelemetry
 Span](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/api.md#span).
-`Sdk.CreateTracerProvider` sets up the OpenTelemetry Sdk, and
-configures it to subscribe to the activities from the source
+OpenTelemetry TracerProvider which is setup at the beginning is
+configured to subscribe to the activities from the source
```

Comment:
Maybe it's just me, but this reads kinda funny. How about: "An OpenTelemetry TracerProvider is configured to subscribe to the activities from the source..."?

Reply:
:D agree. Let me rephrase.